producers-commissioning/src/end-to-end-test/java/uk/gov/moj/sdt/producers/comx/SubmitQueryTest.java

```diff
@@ -1,91 +1,89 @@
-     * Method to call remote submit query endpoint to be tested.
-     */
-    @Test
-    public void testValidRequestForMCOLDefence1() {
-        this.callWebService(SubmitQueryRequestType.class);
-    }
-
-    /**
-     * Method to call remote submit query endpoint to be tested.
-     */
-    @Test
-    public void testValidRequestForMCOLDefence2() {
-        this.callWebService(SubmitQueryRequestType.class);
-    }
-
-    /**
-     * Method to call remote submit query endpoint to be tested.
-     */
-    @Test
-    public void testValidRequestForMCOLDefence3() {
-        this.callWebService(SubmitQueryRequestType.class);
-    }
-
-    /**
-     * Method to call remote submit query endpoint to be tested.
-     */
-    @Test
-    public void testValidRequestForMCOLDefence4() {
-        this.callWebService(SubmitQueryRequestType.class);
-    }
-
-    /**
-     * Method to call remote submit query endpoint to be tested.
-     */
-    @Test
-    public void testValidRequestForMCOLDefence5() {
-        this.callWebService(SubmitQueryRequestType.class);
-    }
-
-    /**
-     * Method to call remote submit query endpoint to be tested.
-     */
-    @Test
-    public void testValidRequestForMCOLDefence6() {
-        this.callWebService(SubmitQueryRequestType.class);
-    }
-
-    @Override
-    protected SubmitQueryResponseType callTestWebService(final SubmitQueryRequestType request) {
-        // Get the SOAP proxy client.
-        ISdtEndpointPortType client = getSdtEndpointClient();
-
-        // Call the specific business method for this text - note that a single test can only use one web service
-        // business method.
-        return client.submitQuery(request);
-    }
-
-    @Override
-    protected JAXBElement<SubmitQueryResponseType> wrapJaxbObject(final SubmitQueryResponseType response) {
-        // Use the provided factor to create a wrapped instance of the response.
-        ObjectFactory objectFactory = new ObjectFactory();
-        return objectFactory.createSubmitQueryResponse(response);
-    }
-}
+package uk.gov.moj.sdt.producers.comx;
+
+import javax.xml.bind.JAXBElement;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import uk.gov.moj.sdt.ws._2013.sdt.sdtendpoint.ISdtEndpointPortType;
+import uk.gov.moj.sdt.ws._2013.sdt.submitqueryrequestschema.SubmitQueryRequestType;
+import uk.gov.moj.sdt.ws._2013.sdt.submitqueryresponseschema.ObjectFactory;
+import uk.gov.moj.sdt.ws._2013.sdt.submitqueryresponseschema.SubmitQueryResponseType;
+
+/**
+ * Test class for end to end web service tests..
+ *
+ * @author Robin Compston
+ */
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+public class SubmitQueryTest extends AbstractWebServiceTest<SubmitQueryRequestType, SubmitQueryResponseType> {
```

Comment:
Does this class need to be annotated with `@SpringBootTest` like RequestBulkFeedbackTest has?

Reply:
it is annotated with @SpringBootTest